### PR TITLE
create new input and textarea

### PR DESCRIPTION
**Pull request by JoshK2**
New input and Textara components

image

### CODE REVIEW

File: src/ui/input/text/text.module.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  color: var(--bit-text-color-heavy, #2b2b2b);
+
+  &.error {
+    background-color: var(--bit-accent-bg, #fdeff2);
+    border: 1px solid var(--bit-accent-color, #e62e5c);
+  }
+  &.success {
+    background-color: var(--bit-accent-bg, #eefaf3);
+    border: 1px solid var(--bit-accent-color, #37b26c);
+  }
```
Comment (KutnerUri) [inline src/ui/input/text/text.module.scss:18]: unify these (`&.error, &.success`) ,
or replace with a single `.accent`?

File: src/ui/input/text/text.module.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.input {
+  outline: 0;
+  padding: 11px 16px;
```
Comment (KutnerUri) [inline src/ui/input/text/text.module.scss:3]: no sizes in default. use em, or extract to presets file

Comment (KutnerUri) [reply src/ui/input/text/text.module.scss:3]: please make a composition of 3 different sizes. we never have just one one size!

File: src/ui/input/text/text.module.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  }
+  &:disabled {
+    cursor: not-allowed;
+    background-color: var(--bit-bg-dent, #f6f6f6);
+    color: var(--bit-text-inactive, #babec9);
+  }
+  &:focus {
+    box-shadow: var(--bit-shadow-raised-low, 0 -1px 1px 0 rgba(0, 0, 0, 0.09), 0 2px 2px 0 rgba(0, 0, 0, 0.23));
+  }
+  &:hover:not(:disabled):not(:focus) {
```
Comment (KutnerUri) [inline src/ui/input/text/text.module.scss:30]: we shouldn't use so many `:not()`, but it's ok

File: src/ui/input/text/text.module.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    border: 1px solid var(--bit-text-inactive, #babec9);
+  }
+}
+
+textarea.input {
+  resize: none;
+}
```
Comment (KutnerUri) [inline src/ui/input/text/text.module.scss:37]: why not put it in the normal level? `.input { resize: none }`

**Review by KutnerUri — CHANGES_REQUESTED** [review]

File: src/ui/input/text/text.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+const getAccent = ({ error, success }: Props) => {
+  return (
+    (error && classNames(colorPalette.error, styles.error)) ||
+    (success && classNames(colorPalette.success, styles.success))
+  );
+};
+
+export function Text({ className, error, success, ...rest }: TextProps) {
+  const accent = getAccent({ error, success });
```
[the remark this answers is not in the document]
Comment (JoshK2) [reply src/ui/input/text/text.tsx:22]: Thanks!

File: src/ui/input/text/text.module.scss (hunk 3 above)
Comment (JoshK2) [reply src/ui/input/text/text.module.scss:30]: if I don't use not(), and its disabled, so when I hover its override some css. (The same for focus) and I don't want this.

File: src/ui/input/text/text.module.scss (hunk 2 above)
Comment (JoshK2) [reply src/ui/input/text/text.module.scss]: done

**Review by KutnerUri — APPROVED** [review]

**Pull request merged**